pkg/scheduler/framework/util.go

```diff
@@ -263,3 +265,25 @@ func (nl *NodeLister) List() ([]*v1.Node, error) {
 	}
 	return nodes, nil
 }
+
+// The state of the k8s prefile is converted to the internal state of the volcano
```

Comment:
please provide the function signature formly.

Reply:
done